spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'John', posts_counter: 30, photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.') }
```

Comment:
- [ ] your tests are failing, kindly change `posts_counter` to `post_counter` as it is how it looks in your schema
![image](https://user-images.githubusercontent.com/66097371/205461299-a1aeabee-52d9-441e-a291-f3218227fdde.png)